### ISSUE TITLE
when playing game pressing windows button opens file explorer

### ISSUE THREAD

**Issue by Leaf533** [valinet/ExplorerPatcher#3390]
when playing game pressing windows button opens file explorer.

can some one please help me, i was mildly inconvenienced

**Comment by pyrates999** [comment]
Please try to install the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Comment by Leaf533** [comment]
didnt work

**Comment by pyrates999** [comment]
what do you mean? were you not able to install it?

**Comment by Leaf533** [comment]
no i installed it but the problem didnt go away

**Comment by pyrates999** [comment]
Usually if you press windows key and E together, that will open up windows explorer.  You might be doing that accidentally?

If you don't need the windows key, you can try to disable it: https://www.makeuseof.com/enable-disable-windows-key/

I've never disabled it, so I can't say if this works or not.

**Comment by Leaf533** [comment]
no, i definitely didnt press windows+E i only pressed the windows button while on an application that was full screened and it opened file explorer for no reason, but its fine ill just uninstall it. thanks for all the help.
